user/user.go

```diff
@@ -83,6 +87,20 @@ type Capabilities struct {
 	ForceMute              bool `ocscapability:"force-mute"`
 	ConversationV2         bool `ocscapability:"conversation-v2"`
 	ChatReferenceID        bool `ocscapability:"chat-reference-id"`
+	ConversationV3         bool `ocscapability:"conversation-v3"`
+	ConversationV4         bool `ocscapability:"conversation-v4"`
+	SIPSupport             bool `ocscapability:"sip-support"`
+	ChatReadStatus         bool `ocscapability:"chat-read-status"`
+	ListableRooms          bool `ocscapability:"listable-rooms"`
+	PhonebookSearch        bool `ocscapability:"phonebook-search"`
+	RaiseHand              bool `ocscapability:"raise-hand"`
+	RoomDescription        bool `ocscapability:"room-description"`
+	DeleteMessages         bool `ocscapability:"delete-messages"`
+	RichObjectSharing      bool `ocscapability:"rich-object-sharing"`
+	ConversationCallFlags  bool `ocscapability:"conversation-call-flags"`
+	GeoLocationSharing     bool `ocscapability:"geo-location-sharing"`
+	ReadPrivacyConfig      bool `ocscapability:"config => chat => read-privacy"`
+	MaxGifSizeConfig       int  `ocscapability:"config => previews => max-gif-size"`
```

Comment:
there are some more, I noticed they are not documented so now I implemented a test for it:
https://github.com/nextcloud/spreed/pull/5664

Reply:
Added them.